src/components/Questionnaire/QuestionnaireForm.tsx

```diff
@@ -77,10 +78,19 @@ export function QuestionnaireForm({
     prefetch: !!questionnaireSlug && !FIXED_QUESTIONNAIRES[questionnaireSlug],
   });
 
-  const { mutate: submitBatch, isProcessing } = useMutation(
-    routes.batchRequest,
-    { silent: true },
-  );
+  const { mutate: submitBatch, isPending: isProcessing } = useMutation({
```

Comment:
let's switch to `isPending` itself. I don't see any purpose for aliasing this.

```suggestion
  const { mutate: submitBatch, isPending } = useMutation({
```

Reply:
@rithviknishad  `isPending: isProcessing` alias was made because `isProcessing` was already in use in sub part of the program 7+ times
```
disabled={isProcessing}
disabled={isProcessing || hasErrors}
```
if required I will change it